keyboards/mechlovin/kanu/kanu.c

```diff
@@ -35,12 +35,3 @@ bool led_update_kb(led_t led_state) {
     return true;
 }
 
-layer_state_t layer_state_set_kb(layer_state_t state) {
```

Comment:
Would prefer if this wasn't removed.  

If anything, setting it to _user, and adding `__attribute__((weak))` to it would be a better solution.